parlai/agents/language_model/language_model.py

```diff
@@ -487,6 +494,15 @@ def shutdown(self):
             self.save(path + '.shutdown_state')
         super().shutdown()
 
+    def receive_metrics(self, metrics_dict):
+        if 'loss' in metrics_dict:
+            if self.best_val_loss is None:
+                self.best_val_loss = metrics_dict['loss']
+            else:
+                if metrics_dict['loss'] > self.best_val_loss:
+                    self.lr *= self.lr_factor
+                    print("Updating learning rate: lr =", self.lr)
```

Comment:
do you want to use loss or something like `opt.get('valid_metric', opt.get('accuracy'))`?

Reply:
nevermind